lib/super_good/solidus_taxjar/api.rb

```diff
@@ -50,6 +50,22 @@ def validate_spree_address(spree_address)
         taxjar_client.validate_address ApiParams.validate_address_params(spree_address)
       end
 
+      def create_customer_for(user)
+        taxjar_client.create_customer ApiParams.customer_params(user.taxjar_customer)
+      end
+
+      def update_customer_for(user)
+        taxjar_client.update_customer ApiParams.customer_params(user.taxjar_customer)
+      end
+
+      def delete_customer_for(user)
+        taxjar_client.delete_customer user.id
+      end
+
+      def nexus_regions
```

Comment:
I know this is a simple method, but could we get a `spec` for it for completeness?

Reply:
Sure, i can create a test for that